openvm/src/lib.rs

```diff
@@ -722,9 +722,9 @@ mod tests {
         .powdr_airs_metrics();
         assert_eq!(machines.len(), 1);
         let m = &machines[0];
-        assert_eq!(m.width, 3701);
+        assert_eq!(m.width, 3541);
         assert_eq!(m.constraints, 506);
-        assert_eq!(m.bus_interactions, 2922);
+        assert_eq!(m.bus_interactions, 3207);
```

Comment:
The increase here seems to come from previously removed bitwise lookups, see the [diff on the Keccak benchmark](https://gist.github.com/georgwiese/fecd4d36246035c433cabb14a5bcd166).